src/Components/GeneralComponents/GeneralWindow/GeneralWindow.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import GeneralCloseButton from '../GeneralCloseButton/GeneralCloseButton';
+
+export default function GeneralWindow(props) {
+  const basicWindowStyle = {
+    backgroundColor: '#ddcc66',
+    textAlign: 'right',
+    padding: '7px',
+    borderRadius: '5px',
+    zIndex: '1',
+  };
+
+  const centeredWindowStyle = {
+    position: 'fixed',
+    left: '50%',
+    top: '50%',
+    transform: 'translate(-50%,-50%)',
+  };
+
+  const rightlyPositionedWindowStyle = {
+    position: 'fixed',
+    right: '7%',
+    top: '50%',
+    transform: 'translate(-50%,-50%)',
+  };
+
+  const composedWindowStyle = Object.assign(
+    basicWindowStyle,
+    props.windowPosition === 'right' ? rightlyPositionedWindowStyle : centeredWindowStyle
```

Comment:
please use distracturing for `props.windowPosition`

Reply:
please use const for the bool and const for the selected window style that is base on this bool.